src/openms/source/ANALYSIS/ID/IDScoreGetterSetter.cpp

```diff
@@ -39,10 +39,11 @@ using namespace std;
 namespace OpenMS
 {
 
-  /** @ingroup getScoresFunctions
-      * @brief For protein groups. Groups are target if at least one protein is target
-      * Therefore it needs an unordered set of decoy accessions to evaluate that.
-    */
+  /**
+   * @ingroup getScoresFunctions
```

Comment:
not sure how important but seems we recently got rid of the * in multi line comments.